File: Service_eHospital_DongNai_A/Implement/HoiChanXaTri_EkipService.cs
namespace Service_eHospital_DongNai_A.Implement
{
    public class HoiChanXaTri_EkipService : BaseService<HoiChanXaTri_Ekip, IHoiChanXaTri_EkipRepository>
    , IHoiChanXaTri_EkipService
    {
    private readonly IHoiChanXaTri_EkipRepository _HoiChanXaTri_EkipRepository;
    public HoiChanXaTri_EkipService(IHoiChanXaTri_EkipRepository HoiChanXaTri_EkipRepository) : base(HoiChanXaTri_EkipRepository)
    {
    _HoiChanXaTri_EkipRepository = HoiChanXaTri_EkipRepository;
    }
    }
    }

File: CRM/src/app/shared/eHospital_DongNai_A/HoiChanXaTri_Ekip.service.ts
import { Injectable } from '@angular/core';
import { HttpClient, HttpParams } from '@angular/common/http';
import { environment } from 'src/environments/environment';
import { HoiChanXaTri_Ekip } from './HoiChanXaTri_Ekip.model';
import { BaseService } from './Base.service';
@Injectable({
    providedIn: 'root'
})
export class HoiChanXaTri_EkipService extends BaseService{
    DisplayColumns001: string[] = ['Save', 'STT', 'HoiChanXaTri_BacSi_Id', 'HoiChanXaTri_Id', 'NhanVien_Id', 'VaiTro_Id'];

    List: HoiChanXaTri_Ekip[] | undefined;
    ListFilter: HoiChanXaTri_Ekip[] | undefined;
    FormData!: HoiChanXaTri_Ekip;
    constructor(public httpClient: HttpClient) {
        super(httpClient);
        this.Controller = "HoiChanXaTri_Ekip";
    }
}

